packages/vats/src/core/startWalletFactory.js

```diff
@@ -15,35 +14,57 @@ const { details: X } = assert;
 // eslint-disable-next-line no-unused-vars
 const startFactoryInstance = (zoe, inst) => E(zoe).startInstance(inst);
 
+const StableUnit = BigInt(10 ** Stable.displayInfo.decimalPlaces);
+
+const PROVISION_ACCT_TEST = 'agoric1muveuvn0nka9g6mx9fs0uxls5pvnmd23376dwq';
```

Comment:
Should this go away before merging?   If not, what is its role in production?

Reply:
it's gone